torch/onnx/_internal/fx/export.py

```diff
@@ -187,118 +274,89 @@ def _export_fx_to_ts(fx_module_with_metadata, opset_version):
-                )
-                assert symbolic_function_group is not None
-                symbolic_fn = symbolic_function_group.get(opset_version)
-                assert symbolic_fn is not None
-                # TODO(wechi): current type checking throws when feeding torch._C.Graph
-                # to symbolic_opset*.py functions, so we need the following wrapper.
-                # After we get rid of TorchScript, we can remove this wrapper.
-                graph_context = jit_utils.GraphContext(
-                    graph=g,
-                    block=g.block(),  # Pointless. Just make linter happy.
-                    opset=opset_version,
-                    original_node=g.insertPoint(),  # Pointless. Just make linter happy.
-                    params_dict={},  # Pointless. Just make linter happy.
-                    env={},  # Pointless. Just make linter happy.
-                )
+
+                # only latest opset version is only supported in atenlib for now
+                symbolic_fn = _ATENLIB_FUNCTIONS.get(exporter_key)
+                assert (
+                    symbolic_fn is not None
+                ), f"Cannot find function for {exporter_key}"
                 # Map FX inputs to ONNX inputs and fill optional inputs with default values.
-                ts_args = _wrap_fx_args_as_ts_args(
-                    graph_context, fx_module_with_metadata, node, fx_name_to_ts_value
+                onnx_args = _wrap_fx_args_as_ts_args(
+                    fx_module_with_metadata, node, fx_name_to_onnxscipt_tensor
                 )
-                # The returned value could be a value of a tuple of values.
-                v = symbolic_fn(graph_context, *ts_args)
+                with evaluator.default_as(tracer):
+                    output: Union[
+                        graph_building.TorchScriptTensor,
+                        Tuple[graph_building.TorchScriptTensor],
+                    ] = symbolic_fn(*onnx_args)
                 assert (
-                    v is not None
-                ), f"Node creates None with target={node.target}, name={node.name}, args={ts_args}"
+                    output is not None
+                ), f"Node creates None with target={node.target}, name={node.name}, args={onnx_args}"
                 # Assign type and shape obtained from FakeTensorProp.
                 # _fill_tensor_types(v, node.meta["val"])
                 # One fx node could produce multiple outputs (e.g., tuple of tensors); in
-                # that case, v is a tuple of TorchScript values.
-                fx_name_to_ts_value[node.name] = v
+                # that case, v is a tuple of TorchScriptTensors.
+                assert isinstance(output, graph_building.TorchScriptTensor)
+                assert isinstance(output, onnxscript.tensor.Tensor)
+                fx_name_to_onnxscipt_tensor[node.name] = output
             elif node.target == operator.getitem and isinstance(node.args, tuple):
```

Comment:
I wonder what's `node.op` for this case? I feel this should be handled just as other regular ops.

Reply:
SG

Reply:
Done